themes/default/content/blog/pulumi-release-notes-m64/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Nov. 17 releases: `dependOn` Helm charts, new Elastic Cloud provider, functions support outputs, set the CLI's default organization"
+date: 2021-11-17T08:00:00-07:00
+draft: false
+allow_long_title: true
+meta_desc: 
+meta_image: meta.png
+authors:
+    - alex-mullans
+tags:
+    - features
+---
+
+It's been another exciting few weeks here at Pulumi! We've caught our breath from [Cloud Engineering Summit]({{<relref "/cloud-engineering-summit">}}) (don't forget to check out the talks if you haven't yet!) and we're back to adding new value and highly-requested fixes across the Pulumi Cloud Engineering Platform. Read on to learn about new providers, new enhancements to the core Pulumi experience, and more!
+
+- Pulumi Registry, Pulumi Packages, & integrations
+  - [`dependsOn` can now await a fully-deployed Helm chart]({{<relref "/blog/pulumi-release-notes-m64#dependson-can-now-await-a-fully-deployed-helm-chart">}})
+  - [New provider: Elastic Cloud]({{<relref "/blog/pulumi-release-notes-m64#new-provider-elastic-cloud">}})
+  - [AWS Native provider supports auto-naming]({{<relref "/blog/pulumi-release-notes-m64#aws-native-provider-supports-auto-naming">}})
+  - [New resources in the AWS Native provider]({{<relref "/blog/pulumi-release-notes-m64#new-resources-in-the-aws-native-provider">}})
+  - [New resources in the Azure Native provider]({{<relref "/blog/pulumi-release-notes-m64#new-resources-in-the-azure-native-provider">}})
+- Pulumi CLI and core technologies
+  - [Functions now support outputs]({{<relref "/blog/pulumi-release-notes-m64#functions-now-support-outputs">}})
+  - [`--json` flag for `pulumi {update|destroy|preview|refresh}`]({{<relref "/blog/pulumi-release-notes-m64#--json-flag-for-pulumi-updatedestroypreviewrefresh">}})
+  - [Python 3.6 support will be deprecated soon]({{<relref "/blog/pulumi-release-notes-m64#python-36-support-will-be-deprecated-soon">}})
+- Pulumi Service & Pulumi.com
+  - [Set the default organization for newly-created stacks]({{<relref "/blog/pulumi-release-notes-m64#new-resources-in-the-azure-native-provider">}})
+
+<!--more-->
+
+## Pulumi Registry, Pulumi Packages, & integrations
+
+### `dependsOn` can now await a fully-deployed Helm chart
```

Comment:
Will I be confused reading this if I'm a user of the more recent `Release` abstraction? For what it's worth, I'm not an avid user, but I am confused about how this behaves with the new resource type ... Maybe worth a mention?

Reply:
This is a great point, thanks. I've added a new note to clarify and also made subtle adjustments throughout to clarify that this is about the `Chart` resource, not the `Release` resource.